src/main/java/uk/ac/cam/cl/dtg/util/mappers/AudienceContextMapper.java

```diff
@@ -0,0 +1,36 @@
+package uk.ac.cam.cl.dtg.util.mappers;
```

Comment:
Since all the mappers affect Isaac DTOs and DOs, would it make sense to move them all to `uk.ac.cam.cl.dtg.isaac.mapper`? Happy to be challenged here.

Reply:
There appear to be a number of _usages_ of the mapper classes in `.segue` as well as `.isaac`, so I'm not sure that would be an accurate placement?
On the other hand, it does look like the actual classes _being_ mapped are all defined in `.isaac`, so I could see an argument either way.
Let me know if the additional information changes your recommendation or if I should go ahead with moving the subfolder to `.isaac`

Reply:
Exactly, because the classes being mapped are all `isaac`, it makes sense to me that they live there. Then `segue` classes can export it from `isaac`, in the same way they're already exporting `isaac.dos` and `isaac.dto`. Hopefully refactoring all this should be quite easy with IntelliJ, as you can move a whole package to a different directory. Just make sure the tests are moved as well.